packages/ui/src/common/model/blocksToTime.ts

```diff
@@ -0,0 +1,16 @@
+import BN from 'bn.js'
+
+import { significantTimeUnit } from './significantTimeUnit'
+
+export function blocksToTime(blocks: BN) {
+  const seconds = blocks.toNumber() * 7
+  if (seconds < 0) {
+    return 'never'
+  }
+  const { count, unit } = significantTimeUnit(seconds)
+  if (unit) {
+    const format = Intl.NumberFormat('en', { style: 'unit', unit, unitDisplay: 'long' })
+    return format.format(count)
+  }
+  return 'less than a minute'
```

Comment:
I was thinking a bit about this. I think that we can have this for now, but maybe use 'short' as in the design?

So here it would be "\< 1 min" or we can just go to seconds :shrug: – it's a coin toss for me which will be better:

*   \< 1 min
*   12 sec
*   now (for 0 s)

But the `"short"` format for sure :)

Reply:
I think "<1 min" is good enough precision, especially that not every block takes exactly the same time to mine

Reply:
Also I'd forego "now" for the exact same reason, especially that it'll roll over to ~14400b/1 day right next block